massa-bootstrap/src/server.rs

```diff
@@ -432,7 +432,10 @@ async fn manage_bootstrap(
     loop {
         match tokio::time::timeout(bootstrap_settings.read_timeout.into(), server.next()).await {
```

Comment:
so at every loop iteration we expect a timeout here ? Doesn't this slow things down a lot ?

Reply:
No we don't except a timeout at every loop iteration. We wait for a message and manage it if we timeout then we stop the connection.

Reply:
yes you're right :)